Cubical/Homotopy/Group/Properties.agda

```diff
@@ -0,0 +1,718 @@
+{-# OPTIONS --safe --experimental-lossy-unification #-}
+{-
+This file contians:
+1. A proof that the equivalence Ωⁿ A ≃ (Sⁿ →∙ A)
+is structure preserving
+
+2. Using the above, the complete group structure on (Sⁿ →∙ A),
+the alternative definition of homotopy groups
```

Comment:
Couldn't this be done using the SIP? Seems like you're doing it manually now?

Reply:
Indeed, I'm not sure if we can do this with the current SIP setup... Can @ecavallo help us investigate this? Maybe we can take a look in person on Monday afternoon?

Reply:
Sure. Just from looking I'm not clear on what exactly you want to sip.

Reply:
Axel and I discussed the code today over Zoom and it seems maybe possible to SIP some parts, but it's not obvious how... Anyway let's just look at it together on Monday after lunch

Reply:
https://github.com/ecavallo/cubical/blob/GysinCleanup/Cubical/Homotopy/Group/Properties.agda
Left a hole for a proof of inverse preservation.

Reply:
Nice! But couldn't one transfer all of the laws in one go? So you define `isWildGroup` as a structure on `RawGroup` and subst it. 

It then doesn't seem necessary to extract the fields as they're only used to define https://github.com/ecavallo/cubical/blob/GysinCleanup/Cubical/Homotopy/Group/Properties.agda#L264 ... Or?